src/multi-input/index.jsx

```diff
@@ -51,14 +69,19 @@ export default function MultiInput({ value, onChange, onFieldChange }) {
 
   return (
     <div className="multi-input">
-      <fieldset>
+      {
+        objectItems && <input type="hidden" name={name} value={JSON.stringify(getItems())} />
+      }
+      <fieldset id={name}>
```

Comment:
What's the intention of adding the id here? I have a feeling we might end up with multiple items with the same id.

Reply:
it's for when you click the "error" validation link - it takes you to `#id`. There is only one as it's added to the fieldset not the field, if we have multiple things with the same `name` on the page then there would be other problems